client/packages/system/src/Patient/ProgramEnrolment/Components/ListView.tsx

```diff
@@ -15,11 +15,13 @@ import {
 import {
   PatientModal,
   ProgramEnrolmentRowFragmentWithId,
+  ProgramEventFragment,
   usePatientModalStore,
   useProgramEnrolments,
 } from '@openmsupply-client/programs';
 import { usePatient } from '../../api';
 import { getStatusTranslation } from '../utils';
+import { encounterEventCellValue } from 'packages/system/src/Encounter/ListView/columns';
```

Comment:
@CarlosNZ forgot to mention this import...

Reply:
Okay, I can change it to `import { encounterEventCellValue } from '@openmsupply-client/system/src/Encounter/ListView/columns';`

Is that okay?

Reply:
Or feel free to just fix it yourself directly on the feature branch :)

Reply:
Yes 

> Okay, I can change it to `import { encounterEventCellValue } from '@openmsupply-client/system/src/Encounter/ListView/columns';`
> 
> Is that okay?

Yes would say so. @mark-prins or export it all to the way to the root `index.ts` in the system package?

Reply:
sorry, I missed commenting on this one.. I try not to, and only import from the root i.e. `from '@openmsupply-client/system'` but sure, the import will work like that.